axelor-bank-payment/src/main/resources/views/BankReconciliation.xml

```diff
@@ -144,6 +147,17 @@
         title="Computed balance in accounting and Ending Balance of the statement should be equal to validate the Statement"
         showIf="computedBalance != movesTheoreticalBalance" colSpan="12" css="label-danger"/>
     </panel>
+    <panel colSpan="12" showIf="corrected" title="Correction">
+      <field name="comments"
+        showIf="statusSelect == 3 || statusSelect == 2 &amp;&amp; comments != null"
```

Comment:
On the status "Under correction", the field "comments" should be displayed even if empty